hoomd/md/pair/pair.py

```diff
@@ -1165,6 +1165,78 @@ def __init__(self, nlist, r_cut=None, r_on=0., mode='none'):
         self._add_typeparam(params)
 
 
+class ExpandedMie(Pair):
+    """Expanded Mie pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list
+        r_cut (float): Default cutoff radius :math:`[\\mathrm{length}]`.
+        r_on (float): Default turn-on radius :math:`[\\mathrm{length}]`.
```

Comment:
These are changing to `default_r_on` and `default_r_cut` now. Please go ahead with this change for this potential (no need to modify others since they are changed on another branch).

Reply:
For pytest checks to pass, I think the 2 branches will have to be merged given these changes?

Reply:
Yes, this is correct. I just merged master into this branch.